### PR TITLE
[Inductor] preserve AliasedLayout on View

### PR DESCRIPTION
**Pull request by jgong5**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #96948

Fix https://github.com/pytorch/pytorch/issues/96728

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/ir.py [hunk 1]
```diff
@@ -3826,12 +3827,7 @@ def create(
     ):
         kernel = "torch.ops.mkldnn._convolution_transpose_pointwise"
         transposed = True
-        (
-            inputs,
-            constant_args,
-            kernel_layout,
-            _,
-        ) = _prepare_convolution_fusion_create(
+        (inputs, constant_args, kernel_layout, _,) = _prepare_convolution_fusion_create(
```
Comment (jgong5) [inline torch/_inductor/ir.py:3830]: lintrunner produced this, not me.

**Review by Chillee — APPROVED** [review]
Can we also add the other test provided? https://github.com/pytorch/pytorch/issues/96728#issuecomment-1469227907

**Comment by jgong5** [comment]
> Can we also add the other test provided? [#96728 (comment)](https://github.com/pytorch/pytorch/issues/96728#issuecomment-1469227907)

Thanks for the note. Added.

**Pull request closed without merging**